configuration/micro_kernel_trait.rst

```diff
@@ -319,6 +319,17 @@ this:
     ├─ composer.json
     └─ composer.lock
 
+As before you can use PHP built-in server:
+
+.. code-block:: bash
+
+    cd web/
+    $ php -S localhost:8000
+
+Then see webpage in browser:
+
+> http://localhost:8000/random/10
```

Comment:
This is markdown syntax. I realize this isn't your fault, as it's wrong above as well. Can you please update both lines to be something like:

```rst
Then visit the webpage in the browser:

    http://localhost:8000/random/10
```

*(actually, I really like to make these links clickable, but not sure what others think about that, so let's not do that in this PR)*